spec/migrations/20220202122417_add_parent_id_to_miq_requests_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_migration
+
+describe AddParentIdToMiqRequests do
+  let(:miq_request) { migration_stub(:MiqRequest) }
+
+  migration_context :up do
```

Comment:
@akhilkr128 simple schema changes like adding a column don't require spec tests, that is more for data migrations.  Just testing that after running the migration you can set a value to the newly created column is really just testing that `ActiveRecord::Migration` is working.